deployment/deploy/deploy.cpp

```diff
@@ -1127,8 +1127,8 @@ IPropertyTree* getInstances(const IPropertyTree* pEnvRoot, const char* compName,
     IPropertyTree* pComponent = &iter->query();
     const char* type = pComponent->queryName();
     if (stricmp(type, "Topology")!=0 && stricmp(type, "Directories")!=0 && 
-        ((!compName && !compType) || (compName && !strcmp(pComponent->queryProp("@name"), compName)) ||
-        (!compName && compType && !strcmp(pComponent->queryProp("@buildSet"), compType))))
+        ((!compName && !compType) || (compName && pComponent->queryProp("@name") && !strcmp(pComponent->queryProp("@name"), compName)) ||
```

Comment:
It's connected to Cassandra wuid support in general, in that it's preventing a crash in configgen if you have added the required section to the environment to enable Cassandra wu support.